README.md

```diff
@@ -1,2 +1,24 @@
 # http-server
```

Comment:
Excellent including the coverage report.  Make sure to change the line 'An echo server...' to what it is for the new branches.